example/src/components/OtherParticipants.tsx

```diff
@@ -47,7 +47,10 @@ export const OtherParticipants = ({
       )}
       {numOfOtherParticipants > 2 ? (
         <View style={styles.textRow}>
-          <Typo variant="label">+ {numOfOtherParticipants - 2} others</Typo>
+          <Typo variant="label">
+            + {numOfOtherParticipants - 2} other
+            {numOfOtherParticipants - 2 > 1 ? 's' : null}
```

Comment:
FYI for such things we have libraries like this one: https://www.i18next.com/translation-function/plurals
but we are not going to add a whole i18n library just for avoiding an if :D so it can stay here